source/Program.cs

```diff
@@ -28,7 +27,7 @@ public static class Program
                 }
             });
 
-            AnnexGame.Start<MainMenu>();
+            AnnexGame.Start<WorldScene>();
```

Comment:
We should load the MainMenu first, rather than the WorldScene., WorldScene is accessible from the MainMenu by hitting Start